infra/gcp/bash/ensure-prod-storage.sh

```diff
@@ -368,6 +373,11 @@ function ensure_all_prod_special_cases() {
             "${serviceaccount}" "k8s-infra-gcr-vuln-scanning" 
     done
 
+    # The release process needs to issue OIDC identity tokens as the
+    # staging signer account. TO be able to do it, we need the IAM creds API
+    color 6 "Ensuring IAM Service Account Credentials API in release process project"
+    ensure_services "${RELEASE_PROCESS_PROJECT}" "iamcredentials.googleapis.com"
```

Comment:
As a non-Googler, my GCP account can't enable this service for  `kubernetes-release-test`. You need to ask a Googler with the correct permissions to do this.

Reply:
I was talking with @cpanato and he noticed that @justaugustus can actuate this too. Stephen, are you able to run the script when it merges?

Reply:
I've invited each of the @kubernetes/sig-release-admins as Owners for `kubernetes-release-test`

ref: https://github.com/kubernetes/k8s.io/blob/8c4b853d705bae02324c18a2519fc5fba14ae7c7/groups/sig-release/groups.yaml#L18-L29

I can actuate, but now all SIG Release Admins should be able to :)

Reply:
Thanks @justaugustus!  Once we migrate this project to community infra, let's make sure we add the same access groups to it as we have on our other projects.